src/ov_utils.cpp

```diff
@@ -109,13 +109,11 @@ std::string getTensorMapString(const std::map<std::string, std::shared_ptr<Tenso
         auto inputInfo = pair.second;
         auto precision = inputInfo->getPrecision();
         auto layout = inputInfo->getLayout();
-        auto shape = inputInfo->getShape();
-        auto effectiveShape = inputInfo->getEffectiveShape();
+        auto shape = inputInfo->getShape_3();
```

Comment:
Keep in mind this (the `getTensorMapString`) is not used anymore, there is `TensorInfo::asString`